modules/fedimint-ln/src/contracts/incoming.rs

```diff
@@ -72,66 +72,19 @@ pub struct FundedIncomingContract {
     pub out_point: OutPoint,
 }
 
-/// Possible outcomes of preimage decryption
-#[derive(Debug, Clone, Eq, PartialEq, Hash, Deserialize, Serialize, Encodable, Decodable)]
```

Comment:
Why do these need to move to `mod`?

Reply:
Actually, I think these best belong in `fedimint-ln/lib`, right?

Reply:
The only goal is to share Preimage impl for incoming and outgoing contracts

Reply:
Discussed this offline and `contracts/mod` seems reasonable for unifying the Preimage definition